alchemy_news/appbuilder/alchemy-news-trend-endpoint.rb

```diff
@@ -0,0 +1,29 @@
+news_url = "http://access.alchemyapi.com/calls/data/GetNews"
```

Comment:
I was going to suggest using HTTPS, but found that the docs now say https://gateway-a.watsonplatform.net/calls/data/GetNews